### ISSUE TITLE
How to disable multiline for class atrributes?

### ISSUE THREAD

**Issue by CRIMSON-CORP** [nebulab/erb-formatter#29]
![image](https://github.com/nebulab/erb-formatter/assets/71358573/575fa27e-205c-4cbe-98a6-71a4e26b32ea)
I really like this formatter, but its kind of a pain for me when class names are broken into their own lines, especially tailwind css classes as in the picture, is there a way to disable and enable this feature

such that classes are aligned like this?
![image](https://github.com/nebulab/erb-formatter/assets/71358573/5051b122-a3d4-49c4-959f-840a9e75527e)

**Comment by corepilotcloud** [comment]
@CRIMSON-CORP did you figure out how to fix this?

**Comment by CRIMSON-CORP** [comment]
> @CRIMSON-CORP did you figure out how to fix this?

No, i went to use another formatter, i didn't see anyone having my issue in all the list of issues, i thought that was the way the package was, so i discarded it

I think I'll reopen the issue

**Comment by elia** [comment]
@corepilotcloud @CRIMSON-CORP I'm working on taking the list of TW classes from somewhere in the right order, but maybe in the meantime I'll just add an option to keep multiple classes on the same line until we reach the max line width boundary, would that help?

**Comment by CRIMSON-CORP** [comment]
Yes that would be awesome 🔥, well done ✅

**Comment by sharkby7e** [comment]
i would love for this to not happen as well! appreciate it!

**Comment by elia** [comment]
Finally fixed in #33! I'm about to publish a new minor 🙌
